lms/envs/common.py

```diff
@@ -114,6 +114,15 @@
 
     # for consistency in user-experience, keep the value of the following 3 settings
     # in sync with the corresponding ones in cms/envs/common.py
+    # .. toggle_name: FEATURES['ENABLE_DISCUSSION_SERVICE']
+    # .. toggle_implementation: DjangoSetting
+    # .. toggle_default: True
+    # .. toggle_description: When True, it will enable the Discussion tab in courseware for all courses. Setting this
+    #   to False will not contain inline discussion components in a course that contains them, if it's present,
+    #   attempting to expand those components will cause errors. So this should only be set to False with an LMS that
+    #   is running courses that do not contain discussion components.
```

Comment:
Consider moving the part about inline discussions to a `toggle_warning`.

Reply:
Added the toggle_warning